Dockerfile.dev

```diff
@@ -0,0 +1,24 @@
+FROM cimg/python:3.9
+ARG VERSION=3.9.1
+ARG NAME=records-mover
+ENV version_env=$VERSION
+ENV name_env=$NAME
+
+USER root
+
+COPY . $HOME/records-mover/
+WORKDIR /records-mover
+
+RUN pyenv install $version_env
+RUN pyenv virtualenv "$version_env" "$name_env"-"$version_env"
+RUN pyenv local records-mover-$version_env
+RUN python -m venv venv \
+    && . venv/bin/activate \
+    && pip install --upgrade pip \
+    && pip install --progress-bar=off -r requirements.txt \
+    && pip install --upgrade --progress-bar=off 'pandas>=1.5.3,<2' \
+    && pip install -e '.[unittest,typecheck]' #\
+    && make citest \
+    && make cicoverage \
+    && make typecheck \
+    && make citypecoverage
```

Comment:
@naswierczek since this is development env, I don't think we should do the checks as the docker build will fail if these fail; sometimes we want to develop on a build that is failing!

Reply:
Good call. I moved this to a docker-compose section.